akka-stream/src/main/resources/reference.conf

```diff
@@ -18,7 +18,7 @@ akka {
       # When this value is left empty, the default-dispatcher will be used.
       dispatcher = ""
```

Comment:
Should we replace the empty string with a concrete string here as well? (Separate PR afterwards, maybe?)

Reply:
That was my intention, missed this one